test-resources/localhost.pem

```diff
@@ -0,0 +1,33 @@
+-----BEGIN CERTIFICATE-----
+MIIFxzCCA6+gAwIBAgIBAjANBgkqhkiG9w0BAQsFADAfMR0wGwYDVQQDDBRQdXBw
+ZXQgQ0E6IGxvY2FsaG9zdDAeFw0xNTAyMTIyMzMwMDdaFw0yMDAyMTIyMzMwMDda
+MBQxEjAQBgNVBAMMCWxvY2FsaG9zdDCCAiIwDQYJKoZIhvcNAQEBBQADggIPADCC
+AgoCggIBAOamci+eYMaXQN12knIb4okn2KLIwvueqznhCIQkIOs4WW0zvkZbzxdx
+yeP7v8bPo1unogpLi9xH3OUm2IhTgeI5ZvaGEKdb5C90rksr8+aIbUghRPvbmz9h
+ZRcM8xNcrTxh2103xOOE2n/Y6Q5yfY7RsbvuMxLWPqRUULbvL587L2SfNOy7hKt3
+CGzJRTRueRWazLn0d2k0QzvH+01pmwVudWsNd9P3Bl2xuDc3zAihlOc5u17VN0Wx
+pxFtSZ7tjoGgiGww5McCPCKvBaEG7FHUcD5yPTcnRuO4d1pTuKsO2FLJ9kv6efD4
+m/ao9peiq0HrbELy8hgFIfnsDPqZJwybnru35B0umW504tQvLY3KsO/2jL/cINYV
+qjbkj2KnjQ/uBJFLdC0GTSlXW6YAcjzUxAesBQ7ssWANIxGhDnKFaFblHID/MOwi
+A0qafbXDl+N0HHrKv99JibxFR8GbLevT0rc11OQStYkPKscftVGdvzVlyr8bAuNv
+PHHfY5jZpydjv1FhWSV8P6rf4PwnBXAwNUdkMoAskpXG9pjB4IgWnZudzU6chLfl
+pTJuPbZbWOEhLl4HdGUTv6dgbvlOsSn9P6p9dgcBxWD0p+lgx14XH8QLxUDKPYx4
+KWypuqyyU9j3Z6N9dFimc/PA9vCtDpwl6ieFzXc1eUDa50LmAukVAgMBAAGjggEX
+MIIBEzA1BglghkgBhvhCAQ0EKFB1cHBldCBSdWJ5L09wZW5TU0wgSW50ZXJuYWwg
+Q2VydGlmaWNhdGUwWgYDVR0RBFMwUYIJbG9jYWxob3N0ggZwdXBwZXSCGnB1cHBl
+dC5jb3JwLnB1cHBldGxhYnMubmV0giBydXNzZWxscy1tYnAuY29ycC5wdXBwZXRs
+YWJzLm5ldDAOBgNVHQ8BAf8EBAMCBaAwIAYDVR0lAQH/BBYwFAYIKwYBBQUHAwEG
+CCsGAQUFBwMCMAwGA1UdEwEB/wQCMAAwHQYDVR0OBBYEFDVYKXwShW3A3itTI4RX
+5y1NMBF8MB8GA1UdIwQYMBaAFP3INLHFz4T1nHnme4Z1OlC2/v5KMA0GCSqGSIb3
+DQEBCwUAA4ICAQAq6HsCnPpwy6gGFBEiZxBwwczVoDHnCk+3d4kmsigVYwEZ7281
+0lFTlZBfSH4cUs9KAoRch+iAvMKOuxiEGmWMwjvjFqHhynWQB/QiQ/GhhAkCzreU
+uraES+Kw+oi2u2o/gKa3rrwzU2hF3pGz0pPUxBI79xniItfFbVCYGWBUY/Xj+bTa
+9cUxc7Bd05MvaNLvmgubN805/LwiK9rJ9jqBiK0CHBnU8EYIUcPPNcbvCtqbCe3m
+XPpYQU2fZaQH6q25JnJvrCnDWS16TtQHXNWE44+px+f0BeOYsoANGx79wfbkWhI8
+2MXp5vLcg4jD77ZvsN8VpokkDr2pbkvF5ASxRc/5a6U7TQLgoAXpo4YOR/pMiHg5
+2qZlU/b2F10fDyiN/P+1oG8ccsPbOnPYzOQOLrv5XqMqDHas0gUyyhvXdhoWa0gW
+n+54H2UC0FZJIF+5vUdL1lh3T/wM0S2JwtO4oFs01WmyamGbfptUJ3mM6D5HW3Mz
+g261cNa7OREaINvgSYCA811BU9ZOoilKegMajPgAzkRduxb9RCmWA84cX9bHf1P9
+LwD+DAy683p2Dbuz78zzhEjHOXF6lhy+cZcyU9B7EZ3Y9pAKaZsdKRbjU0p9wKNS
+Btf9rDXcl8EPAWpGFw8sJjerFRVu8c0mWYwJSsEY0M3TPuhI+b8i+BUspA==
+-----END CERTIFICATE-----
```

Comment:
we've already got test keys in the repo. Are those broken or is it just a matter of using the right ones?